mne/viz/utils.py

```diff
@@ -40,6 +40,12 @@
 COLORS = ['b', 'g', 'r', 'c', 'm', 'y', 'k', '#473C8B', '#458B74',
           '#CD7F32', '#FF4040', '#ADFF2F', '#8E2323', '#FF1493']
 
+_channel_type_prettyprint = {'eeg': "EEG channel",  'grad': "Gradiometer",
+                             'mag': "Magnetometer", 'seeg': "sEEG channel",
+                             'eog': "EOG channel", 'ecg': "ECG sensor",
+                             'emg': "EMG sensor", 'ecog': "ECoG channel",
+                             'misc': "misc. sensors"}
```

Comment:
I'm curious, what is the logic behind your choice of quotation marks? There seems to be some system at work here.

Reply:
Haha no, I think it's effectively random.

Reply:
Same question for your choice channel vs. sensor - for consistency, I'd always use channel (or sensor if you insist). Alternatively, maybe it's sufficient to just call these EEG, sEEG, EOG, ECG, EMG, and ECoG. And you might want to capitalize "Misc. sensors" (and if possible, don't abbreviate).

Reply:
I thought we had a tendency to only call EEG channel, and everything else sensor. I'm not sure though.

Reply:
OK, just wanted to make sure this was not just random.